.gitlab-ci.yml

```diff
@@ -47,7 +47,7 @@ build:
   script:
     - echo "**${CI_PROJECT_DIR}**"
     - rm -rf ${CI_PROJECT_DIR}/install
-    - git clone --single-branch --branch development https://github.com/rest-for-physics/framework.git framework
+    - git clone --single-branch --branch master https://github.com/rest-for-physics/framework.git framework
```

Comment:
Btw, are you sure that these changes has nothing to do with the pipeline failing?

Reply:
Nothing. This is because `development` branch does not exist anymore. The pipeline would fail much earlier.